include/README

```diff
@@ -66,14 +107,13 @@ disabled. They should implement at least a set of types and values documented
 in ODP API specification headers. They are permitted to provide any platform
 specific optimizations (i.e. they might provide types and/or values that map
 directly onto the hardware details, they might provide inline functions to
-speed up execution of the application, etc). These headers MAY use ODP default
+speed up execution of the application, etc.). These headers MAY use ODP default
 ABI headers if they do fit.
 
-Rest of platform-specific headers
----------------------------------
+### Rest of platform-specific headers
```

Comment:
Probably should be "The rest of the platform-specific headers". But "Other platform-specific headers" or "Additional platform-specific headers" might be better.